### PR TITLE
Disable clone button in 'Clone' dialog when the operation is in progress

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: src/components/create-vm-dialog/uiState.js [hunk 1]
```diff
@@ -31,6 +31,18 @@ const CREATE_TIMEOUT = 'CREATE_TIMEOUT';
...
+export function setVmCloneInProgress(name, connectionName) {
+    const vm = {
+        name,
+        connectionName,
+        isUi: true,
+        cloneInProgress: true,
+    };
+
+    store.dispatch(addUiVm(vm));
+    setupCleanupTimeout(name, connectionName, CREATE_TIMEOUT);
```
Comment (martinpitt) [inline src/components/create-vm-dialog/uiState.js:43]: Is that really desirable? Creating a new VM is somewhat predictable, but cloning a VM might take arbitrarily long if there is lots of data to copy?

**Review by martinpitt — COMMENTED** [review]
I mostly wonder about the timeout -- I don't think there is any single static timeout duration which is "correct". But at the same time, with this "do this in the background" it's too much of a "fire and forget".

It might be better to show some "in progress" stub VM which the user can cancel. Honestly, leaving the dialog open with a spinner/progress and an explicit cancel button (i.e. what we have in main) feels a bit safer to me. The bug is just about cancelling not working properly, maybe let's just fix that?

**Comment by KKoukiou** [comment]
> I mostly wonder about the timeout -- I don't think there is any single static timeout duration which is "correct". But at the same time, with this "do this in the background" it's too much of a "fire and forget".
> 
> It might be better to show some "in progress" stub VM which the user can cancel. Honestly, leaving the dialog open with a spinner/progress and an explicit cancel button (i.e. what we have in main) feels a bit safer to me. The bug is just about cancelling not working properly, maybe let's just fix that?

At start I also was leaning to that direction, but cancelling the virt-install process in the middle of a clone, will leave some rubbish behind and a VM with a probably partially copied disk. So I guess it should be cancel and delete the new VM.

**Comment by martinpitt** [comment]
Right, cancel == interrupt + delete the clone sounds good (doesn't make sense to leave anything behind, there is no precious data)

**Comment by KKoukiou** [comment]
The second commit is useless, since we decided to go the other way. So removed it.

**Comment by martinpitt** [comment]
[packit keeps failing](http://artifacts.dev.testing-farm.io/979577e8-db71-42af-8e34-13ecadb2c9d3/), possibly because the branch is 15 commits behind main (and packit doesn't rebase). I pushed a rebase.

**Comment by martinpitt** [comment]
[this packit failure](https://dashboard.packit.dev/results/srpm-builds/31002) is now entirely different -- the PVC of sandcastle is full, quota is exceeded @TomasTomecek

**Review by martinpitt — APPROVED** [review]

**Comment by TomasTomecek** [comment]
you are working during weekend! :) :O

sorry about the failures: we are planning to address the issues which resulted in the outage, so they would not happen in future again

**Pull request merged**